### ISSUE TITLE
[Resummonable Wither Storm] an end game option for the wither storm

### ISSUE THREAD

**Issue by SSgamingYT** [nonamecrackers2/crackers-wither-storm-mod#481]
basically a config option that makes you need to craft a command block stabilizer and place it in the middle, then make this structure around it:

![images](https://user-images.githubusercontent.com/106631203/184236399-b96a4ad9-3a9c-46c6-9c3d-c0ac089652ac.jpg)

then u get a survival command block that you can mine and place with a couple of other functionalities, but the main one is to build the wither storm structure and then you continue but with better gear for the occasion. just a thought!

TL;DR
make the wither storm mod have an end game config option to make it so you need a command block stabilizer with a structure around it (shown above) then you add that in the middle of the wither structure to create the storm

**Comment by watersofnazareth** [comment]
The Wither Storm is going to be resummonable in the future

**Comment by nonamecrackers2** [comment]
I won't be making a command block stabilizer however I do plan on making the WS resummonable.

**Comment by Asteroid4** [comment]
I do like the idea of a resummonable wither storm!

**Comment by SSgamingYT** [comment]
bet try making it have the setup like redstonia

**Comment by SSgamingYT** [comment]
the command block stabilizer was just a block in the machine, the machine itself is just from story mode redstonia

**Comment by SSgamingYT** [comment]
just to make it more expensive to get

**Comment by NinjaSam5399** [comment]
Well maybe if you defeat the wither storm you get to make a command block stabilizer with withered nether star.

**Comment by Krypton6167** [comment]
maybe when the witherstorm dies you can use the withered nether star in the middle of a crafting table and orange/yellow concrete around it to craft a command block.

**Comment by nonamecrackers2** [comment]
Implemented in 2.2/3.2
